keepassxc-browser/background/keepass.js

```diff
@@ -1112,7 +1112,9 @@ keepass.decrypt = function(input, nonce) {
 keepass.enableAutomaticReconnect = function() {
     // Disable for Windows if KeePassXC is older than 2.3.4
     if (!page.settings.autoReconnect
-        || (navigator.platform.toLowerCase().includes('win') && !keepass.compareVersion('2.3.4', keepass.currentKeePassXC))) {
+        || (navigator.platform.toLowerCase().includes('win')
+            && keepass.currentKeePassXC !== ''
```

Comment:
I think this is good to go, but perhaps it's worth relaxing this check to anything that evaluates to false (null, undefined etc.).

Reply:
I'm not sure if that extra check is necessary here because the variable is defined to an empty string by default, and it's only set using `keepass.setcurrentKeePassXCVersion()`. So the possible check should be made to that function instead so the comparison can remain in a simpler form.

Reply:
I would generate take advantage of weak typing where possible, but I'm fine with either solution. Whatever suits you.

Reply:
Or maybe it's just enough to use `!keepass.currentKeePassXC` after all. It should cover all false situations, including null and undefined.

Reply:
That's what I was referring to.